config/geoip_field_mapping.yaml

```
@@ -3,6 +3,31 @@ SrcIP:
   - '%ClientAddress%'
   - '%SourceAddress%'
   - '%SourceIp%'
+  - '%UserDataAddress%'
+  - '%UserDataParam3%'
+
 TgtIP:
   - '%DestinationIp%'
-  - '%DestAddress%'
\ No newline at end of file
+  - '%DestAddress%'
+
+Filter:
+  - Microsoft-Windows-TerminalServices-LocalSessionManager/Operational:
+    - 21 # RDS Sess Logon %UserDataAddress%
+    - 22 # RDS Sess Start (Noisy) %UserDataAddress%
+    - 24 # RDS Sess Disconnect %UserDataAddress%
+  - Microsoft-Windows-TerminalServices-RemoteConnectionManager/Operational:
+    - 1149 # RDS Logon %UserDataParam3%
+  - Security:
+    - 4624 # Logon Success %IpAddress%
+    - 4625 # Logon Failure %IpAddress%
+    - 4648 # Explicit Logon %IpAddress%
+    - 4768 # Kerberos TGT Requested %IpAddress%
+    - 4769 # Kerberos Service Ticket Requested %IpAddress%
+    - 4778 # RDP Session Reconnect %ClientAddress%
+    - 4779 # RDP Session Disconnect %ClientAddress%
+    - 4825 # RDP Logon Failed %ClientAddress%
+    - 5140 # A network share object was accessed %IpAddress%
+    - 5145 # NetShare File Access %IpAddress%
+    - 5156 # The Windows Filtering Platform has allowed a connection %SourceAddress%
+  - Sysmon: 
```

@YamatoSecurity  Sysmon is channnel abbrevation. Doesn't it make the following SUGGESTION content when combined with the other descriptions?

```suggestion
  - Microsoft-Windows-Sysmon/Operational: 
```

---

Thank you! I forget it was an abbreviation.